api/app/weather_models/precip_rdps_model.py

```diff
@@ -34,10 +34,10 @@ async def compute_and_store_precip_rasters(current_time: datetime):
     """
     async with get_client() as (client, bucket):
         for hour in range(0, 24):
-            timestamp = current_time + timedelta(hours=hour)
-            (precip_diff_raster, geotransform, projection) = await generate_24_hour_accumulating_precip_raster(timestamp)
-            key = f"weather_models/{ModelEnum.RDPS.lower()}/{current_time.date().isoformat()}/" + compose_computed_precip_rdps_key(
-                current_time, current_time.hour, hour, SourcePrefix.COMPUTED
+            accumulation_timestamp = current_time + timedelta(hours=hour)
+            (precip_diff_raster, geotransform, projection) = await generate_24_hour_accumulating_precip_raster(accumulation_timestamp)
+            key = f"weather_models/{ModelEnum.RDPS.lower()}/{accumulation_timestamp.date().isoformat()}/" + compose_computed_precip_rdps_key(
```

Comment:
We'll want to keep the `current_time` iso formatted string for this part of the key, otherwise we'll have model run data going into the next day.

Reply:
Depending what you mean by 'model run data going into the next day', we always had data going into the next day. That's what I was attempting to convey with the naming convention in [this teams message](https://teams.microsoft.com/l/message/19:f12682bf308941b7a482f5057d6df123@thread.tacv2/1722296862479?tenantId=6fdb5200-3d0d-4a8a-b036-d3685e359adc&groupId=23431815-15c2-4928-9839-d8878e46b956&parentMessageId=1721769458438&teamName=External%3A%20Predictive%20Services&channelName=talk%20nerdy%20to%20me&createdTime=1722296862479). Maybe we all need to talk naming convention one more time?

Reply:
Previously:

These 2 inputs:
'2024-07-23/12/precip/CMC_reg_APCP_SFC_0_ps10km_2024072312_P005.grib2'
'2024-07-23/12/precip/CMC_reg_APCP_SFC_0_ps10km_2024072312_P029.grib2'
 
Result in:
'2024-07-24/12/precip/COMPUTED_reg_APCP_SFC_0_ps10km_2024072412_P-10.tif'

Reply:
My words weren't great, code is probably easier, I put up a branch here that adds a failing test for what I was trying to convey: https://github.com/bcgov/wps/tree/task/model-run-path-bug, here's the diff against your branch: https://github.com/bcgov/wps/compare/task/rdps-24hr-precip-output...task/model-run-path-bug?expand=1

If you pull that branch down and run that test you'll see that the date part of the path prefix goes into the next day.

Reply:
I think there might be some confusion around this due to the original implementation where the following happened:
1. Download most recent RDPS model run data
2. Create the Computed RDPS 24 hour accumulated precip using RDPS model run data from 24 hours ago. The computed files were supposed to be named with today's timestamp.

Now it looks like we:
1. Download most recent RDPS model run data
2. Create Computed RDPS 24 hour accumulated precip using the data we just downloaded. This means that the computed files should be named with a timestamp 24 hours into the future according to my brain's way of thinking about this.

Reply:
I think there is an issue here though, we start doing the accumulated precip calculations from the current time, not from the start time of the model run. I think we should be doing:
CMC_reg_APCP_SFC_0_ps10km_2024073012_P024.grib2 - None =  CMC_reg_APCP_SFC_0_ps10km_2024073112_P000.grib2

CMC_reg_APCP_SFC_0_ps10km_2024073012_P025.grib2 - CMC_reg_APCP_SFC_0_ps10km_2024073012_P001.grib2 =  CMC_reg_APCP_SFC_0_ps10km_2024073112_P001.grib2

CMC_reg_APCP_SFC_0_ps10km_2024073012_P026.grib2 - CMC_reg_APCP_SFC_0_ps10km_2024073012_P002.grib2 =  CMC_reg_APCP_SFC_0_ps10km_2024073112_P002.grib2

Reply:
Ah I see, I thought we wanted to keep the original naming schema outlined by RDPS.

Reply:
> I think there might be some confusion around this due to the original implementation where the following happened:
> 
> 1. Download most recent RDPS model run data
> 2. Create the Computed RDPS 24 hour accumulated precip using RDPS model run data from 24 hours ago. The computed files were supposed to be named with today's timestamp.
> 
> Now it looks like we:
> 
> 1. Download most recent RDPS model run data
> 2. Create Computed RDPS 24 hour accumulated precip using the data we just downloaded. This means that the computed files should be named with a timestamp 24 hours into the future according to my brain's way of thinking about this.

Hmm this is counter to the RDPS naming scheme that I thought we were going to keep.

Reply:
I know we were talking about not wanting to do date math, but it would actually be quite easy to do a different naming convention without having to do date math using the `accumulation_timestamp`, and it might be easier for us to grab the data we want that way? 

That timestamp has the end date & time that we're accumulating to, so an output could easily look like:
`weather_models/rdps/2024-07-30/12/precip/COMPUTED_reg_APCP_SFC_0_ps10km_20240730_21z.tif`
for inputs
`CMC_reg_APCP_SFC_0_ps10km_2024072912_P009.grib2`
`CMC_reg_APCP_SFC_0_ps10km_2024072912_P033.grib2`

Just throwing it out there as an idea for dev chat, feel free to throw it right back

Reply:
**Final naming convention:**
`weather_models/rdps/2024-07-30/12/precip/COMPUTED_reg_APCP_SFC_0_ps10km_20240730_021z.tif`